doc/search-api.md

```diff
@@ -46,176 +59,190 @@ The parameters supported are:
-   but it is good practice to set this to only the fields you actually want
-   information on (doing this will normally increase performance).
+### Filter parameters
+You can filter search results using any of the filter_<field name> or reject_<field name> query parameters.
 
- - ab_tests: a/b test with selected variant type. This allows test to be configured
-   from upstream apps.
+- filter is used to only returns documents that match a value
+- reject excludes documents that don't match a value
 
-   Each a/b test name should be followed by a ':' and then the variant type to
-   be used. If multiple a/b test setting are being passed in they should be
-   comma separated.
+Multiple values may be given, and filters may be specified for multiple fields at once. The filters are grouped by field name; documents will only be returned if they match all of these filter groups, and they will be considered to match a filter group if any of the individual filters in that group match (ie, only one of the values specified for a field needs to match, but all fields with any filters specified must match at least one value).
 
-   No validation is done to ensure the a/b test name provided is current implemented.
+The special value `_MISSING` may be specified as a filter value - this will match documents where the field is not present at all.
 
-## Examples
+For string fields, values are the field value to match.
 
-For example:
+For date fields, values are date ranges. These are specified as comma separated lists of key:value parameters, where key is one of `from` or `to`, and the value is an ISO formatted date (with no timezone). UTC is assumed for all dates handled by rummager. Date ranges are inclusive of their endpoints.
 
-    /search.json?
-     q=foo&
-     start=0&
-     count=20&
-     order=-public_timestamp&
-     filter_organisations[]=cabinet-office&
-     filter_organisations[]=driver-vehicle-licensing-agency&
-     filter_section[]=driving&
-     facet_organisations=10&
-     ab_tests=test1:B,test2:A
+For example: `from:2014-04-01 00:00,to:2014-04-02 00:00` is a range for 24 hours from midnight at the start of April the 1st 2014, including midnight that day or the following day.
 
-Returns something like:
+Currently, it is not permitted to specify multiple values for a date field filter.
+
+Only some fields can be filtered on - an HTTP 422 error will be returned if the requested field is not a value sort field.
+
+If a filter and a reject are specified for the same field, an HTTP 422 error will be returned. However, it is valid to specify a reject for some fields and a filter for others - documents will be required to match the criteria on both fields.
+
+### Aggregation parameters
+
+Aggregations look at all the values of a a field and count up the number of times each one appears in documents matching the search.
+
+You can include facets in any search by specifying one of the facet_<field name> query parameters.
```

Comment:
do we want to mention that this name is the old style and that we will be gradually moving to use `aggregate_*` which is the new terminology?

especially considering that is what you used in your example.

It might also be worth mentioning the returned data has different keys facet/aggregate